django/contrib/admin/static/admin/css/base.css

```
@@ -626,22 +630,53 @@ ul.messagelist li {
     display: block;
     font-weight: 400;
     font-size: 0.8125rem;
-    padding: 10px 10px 10px 65px;
-    margin: 0 0 10px 0;
-    background: var(--message-success-bg) url(../img/icon-yes.svg) 40px 12px no-repeat;
-    background-size: 16px auto;
+    padding: 0.625rem 0.625rem 0.625rem 4.063rem;
```

Hear me out… we could be 0.008px off because of this!

```suggestion
    padding: 0.625rem 0.625rem 0.625rem 4.0625rem;
```